easybuild/easyconfigs/m/Mesa/Mesa-19.0.1-GCCcore-8.2.0.eb

```diff
@@ -59,6 +59,10 @@ configopts += " -Dgbm=false -Dglx-direct=false -Dgallium-drivers='swrast,swr' -D
 configopts += " -Dllvm=true -Dshared-llvm=true"
 configopts += " -Dlibunwind=true -Dglx=gallium-xlib"
 
+# also install header files located include/GL/internal/
+# we can't enable both DRI and Gallium drivers, but we can provide the DRI header file (GL/internal/dri_interface.h)
+postinstallcmds = ["cp -a %(builddir)s/mesa-%(version)s/include/GL/internal/ %(installdir)s/include/GL"]
```

Comment:
Please remove the / after internal. Ending with / usually implies that one missed the ending "." which does not result in the same thing. Same thing below.

Reply:
I don't understand the issue.

I've always used a trailing `/` to make sure I'm copying a directory rather than a file...

In which context can using a trailing `/` be wrong here?

Reply:
Technically it works, but "cp /tmp/a/b/ /tmp/c/d " looks like you want to copy the content of b to d, not to copy the directory b to d i.e. it looks like what you intended to write is "cp /tmp/a/b/. /tmp/c/d" which is quite different.

And it gets even more confusing if the destination has the same name as the source, "cp /tmp/a/b/ /tmp/c/b".

The most distinct way to write it is "cp /tmp/a/b /tmp/c/d/."

And if you really want to make sure it is a directory you copy then this is the way to be really explicit.

mkdir /tmp/c/d/b; cp -a /tmp/a/b/. /tmp/c/d/b/.

Reply:
fixed in 6c8c9f8ba7